docs/spock_primer.adoc

```diff
@@ -149,19 +150,19 @@ The picture on the right shows how blocks map to the conceptual phases of a feat
 special role, which will be revealed shortly. But first, let's have a closer look at the other blocks.
 --
 
-==== Setup Blocks
+==== Given Blocks
 
 [source,groovy]
 ----
-setup:
+given:
 def stack = new Stack()
 def elem = "push me"
 ----
 
-The `setup` block is where you do any setup work for the feature that you are describing. It may not be preceded by
-other blocks, and may not be repeated. A `setup` block doesn't have any special semantics. The `setup:` label is
-optional and may be omitted, resulting in an _implicit_ `setup` block. The `given:` label is an alias for `setup:`,
-and sometimes leads to a more readable feature method description (see <<specs-as-doc,Specifications as Documentation>>).
+The `given` block is where you do any configuration for the feature that you are describing. It may not be preceded by
```

Comment:
I think `setup work` is better than `configuration`

Reply:
Change made.